Services/Authentication/classes/class.ilAuthPlugin.php

```diff
@@ -44,5 +44,13 @@ public function getSlotId()
 	{
 		return 'authhk';
 	}
+
+	/**
+	 * Object initialization done by slot.
+	 */
+	protected final function slotInit()
```

Comment:
all other plugin abstract classes implements this method and as `final`, so also the auth plugin needs to do this (probably forgot then ...)

better would be to remove this method completed, because it's make no sense and the method is always empty (except for one unnecessary require in `ilUDFClaimingPlugin` and `ilAdvancedMDClaimingPlugin`

yes, existing plugins would be incompatible, but I think, it's not bad, because according plugins.ilias.de, no such plugin exists

to be safe anyway, the `final` could exceptionally removed and the should still working